### PR TITLE
Complete challenge

### PR DESCRIPTION
**Pull request by ghost**
Submit for review

### CODE REVIEW

File: challenge.rb [hunk 1]
```diff
@@ -1,26 +1,32 @@
...
 end
 
 def fetch_the_first_two(input)
   #implement your solution here
+  input.take(2)
 end
 
 def fetch_CD_animals(input)
   #implement your solution here
+  input.select{|s| s.downcase.chars[0] == 'c' || s.downcase.chars[0] == 'd'}
```
Comment (sjreich) [inline challenge.rb]: You can simplify this with a regular expression.  Google `rubular` for a nice tool for this.

**Comment by MadKeys** [comment]
I used a regular expression to simplify the fetch_cd_animals method.

File: challenge.rb [hunk 2]
```diff
@@ -1,26 +1,32 @@
...
 end
 
 def fetch_the_first_two(input)
   #implement your solution here
+  input.take(2)
 end
 
 def fetch_CD_animals(input)
   #implement your solution here
+  input.select{|s| s.downcase =~ /(c|d)[a-z]+/}
```
Comment (sjreich) [inline challenge.rb]: Cool - this is on the right track.  But this regex will *capture* a 'c' or 'd' which is followed by at least one other letter - doesn't matter what comes before the 'c' or 'd'.  But we just want to *match* (not capture) an expression that *starts with* 'c' or 'd'.  I'd say give it one more shot, or if you really want I can just say what I've got in mind.

File: challenge.rb [hunk 3]
```diff
@@ -1,26 +1,32 @@
 
 def capitalize_each_string(input)
   #implement your solution here
+  input.map {|s| s.capitalize}
```
Comment (sampriddy) [inline challenge.rb]: Would input.map(&:capitalize) also work?

File: challenge.rb [hunk 4]
```diff
@@ -1,26 +1,32 @@
...
 end
 
 def fetch_the_first_two(input)
   #implement your solution here
+  input.take(2)
 end
 
 def fetch_CD_animals(input)
   #implement your solution here
+  input.select{|s| s.downcase =~ /^[cd][a-z]+/}
```
Comment (sampriddy) [inline challenge.rb]: Looks good to me. What I'm about to say is nitpicky, but you can skip needing to downcase by adding the _i_ option to the end of the regex after the final / to tell it to be case insensitive.

**Pull request closed without merging**